x-pack/plugins/monitoring/common/http_api/elasticsearch_settings/put_elasticsearch_settings_collection_enabled.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+
+export const putElasticsearchSettingsCollectionEnabledResponsePayloadRT = rt.type({
+  // TODO: add payload entries
```

Comment:
maybe this type here https://github.com/elastic/kibana/blob/main/x-pack/test/api_integration/apis/monitoring/elasticsearch_settings/set_collection_enabled.js#L40?

Reply:
As mentioned above, these are explicitly left as placeholders for the client-side TS conversion.